src/main/java/core/basesyntax/List.java

```diff
@@ -13,7 +13,7 @@
 
     T remove(int index);
 
-    T remove(T element);
+    T remove(T element) throws NoSuchMethodException;
```

Comment:
do not change this interface
```suggestion
    T remove(T element);
```